README.md

```diff
@@ -107,12 +135,9 @@ of patent rights can be found in the PATENTS file in the same directory.
 As of April 5, 2017, Parse, LLC has transferred this code to the parse-community organization, and will no longer be contributing to or distributing this code.
 
  [docs]: http://docs.parseplatform.org/ios/guide/
- [blog]: http://blog.parse.com/
```

Comment:
No blog link?

Reply:
I removed it as it may not contain any relevant information anymore about this repo. This seemed to be an artifact of the old parse.com where some articles would be pushed out about new SDK features or demo usages.

Reply:
That's fine then. I want to get back to a point where we consistently put up posts about what we're doing, like coalescing all the ObjC frameworks 😉 .

Reply:
yeah, I'll write something this week, when deprecating / archiving the other repos.